TRANSLATING.md

```diff
@@ -67,5 +67,34 @@ In order to keep consistency in translations, every language should have a trans
 
 -   [Portuguese](https://docs.google.com/spreadsheets/d/1HUa3BO2kwukhX4EXQ-1blXeP5iueUdM23OwDRpfarDg/edit?usp=sharing)
 
+## Merging
+
+Aiming at a more comprehensive git history, pull requests will be squashed with a commit message outlining:
+  - What file(s)/page(s) the PR translated (preferably, 1 PR per file)
+  - What training material the translated file(s)/page(s) are part of
+  - What language it was translated into
+
+This practice gives more freedom/space for committers to get into more detail in their commit title/message without having to add the info above. For example:
+
+```
+PR: Translate to Brazilian Portuguese the Tower section of the basic training
```

Comment:
I thought of making it a code block to make it explicit that this would go in the git commit message, but maybe it's confusing. I will do as you suggested 😄

Reply:
Ops, my bad. I thought of this for the second block but did to both 🙄 😆